charts/eks/templates/statefulset.yaml

```diff
@@ -0,0 +1,516 @@
+{{- if and (not .Values.headless) .Values.pro .Values.embeddedEtcd.enabled }}
```

Comment:
Instead of creating a separate file here, can we merge that into the syncer.yaml?